packages/shadcn/test/CheckboxesWidget.test.tsx

```diff
@@ -0,0 +1,35 @@
+import CheckboxesWidget from '../src/CheckboxesWidget';
+import renderer from 'react-test-renderer';
+import { makeWidgetMockProps } from './helpers/createMocks';
```

Comment:
Can you update all imports to make the absolute imports first
```suggestion
import renderer from 'react-test-renderer';
import CheckboxesWidget from '../src/CheckboxesWidget';
import { makeWidgetMockProps } from './helpers/createMocks';
```